fsspec/caching.py

```diff
@@ -597,7 +597,7 @@ def __init__(
 
             self.data = dict(zip(offsets, blocks))
         else:
-            self.data = data
+            self.data = dict()
```

Comment:
```suggestion
            self.data = {}
```
Apologies for my incorrect feedback before.